src/traces/scattersmith/hover.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var scatterHover = require('../scatter/hover');
+
+function hoverPoints(pointData, xval, yval, hovermode) {
```

Comment:
The `hoverPoints` function here is exactly the same as the one found in `scatterpolar`. Could we remove it from here and reuse the other one in `scattersmith/index.js`?

Reply:
It is not quite the same--the `makeHoverPointText()` function is different between the two modules, which means I can't just import the code from the `scatterpolar`. If I could parameterize `makeHoverPointText()` then this would be easy. Do you have any suggestions?